cric-exchange-app-server/src/entity/User.ts

```diff
@@ -0,0 +1,15 @@
+import {Column, PrimaryGeneratedColumn, Entity} from "typeorm";
+
+@Entity()
+export class User {
+
+    @PrimaryGeneratedColumn()
```

Comment:
Wow! Is this like an auto-incrementing id field?

Reply:
Mind === blown